src/main/java/core/basesyntax/JavaDateTimeApi.java

```diff
@@ -19,7 +23,7 @@
      *                 В любом другом случае бросить DateTimeException
      **/
     public String todayDate(DateTimePart datePart) {
-        return "Today";
+        return LocalDate.now().toString();
```

Comment:
please refactor this method, because it always returns same value, and we need our result to be different depending on input parameters + you never throw exception here 
you can use switch case here

Reply:
sorry, fixed.